app/models/type/attributes.rb

```diff
@@ -80,6 +81,7 @@ def all_work_package_form_attributes(merge_date: false)
 
       OpenProject::Cache.fetch('all_work_package_form_attributes',
                                *wp_cf_cache_parts,
+                               EXCLUDED.count,
```

Comment:
https://github.com/fastruby/fast-ruby?tab=readme-ov-file#arraylength-vs-arraysize-vs-arraycount-code

```suggestion
                               EXCLUDED.length,
```

or if feeling idiomatic:

```suggestion
                               EXCLUDED.size,
```

Reply:
Well, I'll change it, but it's not really impactful here.

Reply:
I've been looking to add cops for some of these fastruby findings. Marked it to start as of now :)

Reply:
Yeah I have a love/hate relation with fastruby advices.
Sometimes it really help, like advising `list << item` instead of `list += item` in a loop in some code being used a lot. That's a huge saving.
And also it helps knowing the most efficient ways to write faster ruby code.
And sometimes it's just annoying because it keeps suggesting low-level enhancements that are largely neglectable compared to a bad N+1 issue.